Header  header
string  mac                 # MAC-address Wifi card
string  access_point        # accesspoint
int64   signal              # current signal strength
int64   signal_avg          # average signal strength
int64   rx_packets          # received bytes
int64   tx_packets          # tranceived bytes
float64 CPU_performance     # performance all Cores
float64 CPU_temperature     # Temperature
float64 pingtime            # Response time from ping

